app/controllers/planning_application/consultations_controller.rb

```diff
@@ -46,6 +46,14 @@ def destroy
       end
     end
 
+    def send_neighbour_letters
+      return if consultation.blank?
+
+      consultation.neighbours.reject(&:letter_created?).each do |neighbour|
+        LetterSendingService.new(neighbour, consultation.neighbour_letter_text).deliver!
+      end
+    end
```

Comment:
Shall we move it on a Job?

Reply:
I think `deliver_later!` would work here, right? I'm not super familiar with ActiveJob but yes, we're likely to want this to send quite a few at once and so we don't want it delaying the page load.

Reply:
Oh actually that's for mailer classes.
I think in the job class you can just add `queue_as :low_priority`